### PR TITLE
Host

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

File: src/main/java/com/ft/message/consumer/proxy/MessageQueueProxyServiceImpl.java [hunk 1]
```diff
@@ -68,7 +71,10 @@ public void destroyConsumerInstance(URI consumerInstance) {
     public List<MessageRecord> consumeMessages(URI consumerInstace) {
```
Comment (BerniVarga) [inline src/main/java/com/ft/message/consumer/proxy/MessageQueueProxyServiceImpl.java]: Variable name misspelled.

Comment (catalin-ilea) [inline src/main/java/com/ft/message/consumer/proxy/MessageQueueProxyServiceImpl.java]: Done

**Comment by Nesze** [comment]
lgtm

**Comment by BerniVarga** [comment]
+1

**Pull request merged**